src/App.js

```
@@ -2,6 +2,17 @@ import logo from './logo.svg';
 import './App.css';
 
 function App() {
+  const a = 1
+  let fun = () => {
+    a++;
+  }
+
+  const b = 1;
+  b++;
```

Biome (2.1.2)

[error] Can't assign b because it's a constant.

This is where the variable is defined as constant.

Unsafe fix: Replace const with let if you assign it to a new value.


(lint/correctness/noConstAssign)